UserInterface.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace Evolution7\BugsnagBundle;
+
+interface UserInterface
+{
+
+    /**
+     * The Method must return an array with all userdata that should be sent to
+     * Bugsnag.
+     *
+     * Note: the array keys "id", "name" and "email" will be searchable!
+     *
+     * @return array
+     */
+    public function get();
```

Comment:
i think this method name is not very descriptive. Can you update it to "toArray" (it's pretty standard) or a more descriptive name?

The key point is that the developer should be able to tell what a method returns without looking at the PHPDoc.